Formula/c/cpuinfo.rb

```diff
@@ -0,0 +1,39 @@
+class Cpuinfo < Formula
```

Comment:
Given that the CLI tool is named `cpu-info`, and that there is already [a cask named `cpuinfo`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/c/cpuinfo.rb), it might be better to name this `cpu-info`… I'm genuinely unsure.

Reply:
to me, `cpuinfo` sounds better than `cpu-info`.